main/test/ca/uwaterloo/flix/language/phase/TestTyper.scala

```diff
@@ -1116,19 +1116,6 @@ class TestTyper extends AnyFunSuite with TestUtils {
     expectError[TypeError](compile(input, Options.TestWithLibNix))
   }
 
-  test("TestCaseSetAnnotation.04") {
-    val input =
-      """
-        |restrictable enum Color[s] {
-        |    case Red, Green, Blue
-        |}
-        |
-        |// Wrong minus parsing
```

Comment:
I removed this test since, as it says, it is verifying that we have the wrong parsing